SU-SWS/stanford_publication
# D8CORE-4012 and D8CORE-4072: Tweaks to spacing for the publications

# READY FOR REVIEW

# Summary
- Fixing the alignment of the cards on the More Publications so the the button doesn't float up.
- Fixing the padding between titles and the type on the node page.

# Needed By (Date)
- 4/26

# Urgency
- med

# Steps to Test

1. Pull in this change.
2. Clear caches.
2. Notice the additional space after the top topic above the header. The padding-bottom at the top type should be MS1/27px between topic and header.
2. Notice the change in the padding-bottom in the More Publication cards. It should be MS-1/12px.
3. Make sure you have 3 publications. Look at the cards at the bottom of a node. There should be two cards and the button for more articles should fall below the cards. In the current version of the site, the button floats up to the card row. 
<img width="1471" alt="Screen Shot 2021-04-07 at 9 09 58 AM" src="https://user-images.githubusercontent.com/1187335/115079879-b30ad480-9eb6-11eb-9ad2-dd2f4876997d.png">
4. It should look like this:
<img width="1664" alt="Screen Shot 2021-04-16 at 9 09 06 AM" src="https://user-images.githubusercontent.com/1187335/115080021-ebaaae00-9eb6-11eb-812d-1d29f449f4bd.png">

# Affected Projects or Products
- Stanford Publications

# Associated Issues and/or People
- D8CORE-4012
- D8CORE-4072


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
